src/main/kotlin/io/github/inductiveautomation/kindling/idb/tagconfig/model/NodeStatistics.kt

```diff
@@ -0,0 +1,115 @@
+package io.github.inductiveautomation.kindling.idb.tagconfig.model
+
+import kotlinx.serialization.json.JsonArray
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.JsonPrimitive
+import kotlinx.serialization.json.boolean
+import kotlinx.serialization.json.jsonObject
+import kotlinx.serialization.json.jsonPrimitive
+
+class NodeStatistics(private val node: Node) {
+    // Tag Type
+    val isUdtDefinition: Boolean
```

Comment:
TagTypes are mutually exclusive - maybe create an enum here?

Reply:
I was thinking of these booleans as being separate statistics from each other, so they had to have different members for each one. I'll have to look into how this might change the overall structure of the implementation